storage/ocp_recommendations_storage.go

```diff
@@ -242,8 +246,10 @@ type OCPRecommendationsDBStorage struct {
 func NewOCPRecommendationsStorage(configuration Configuration) (OCPRecommendationsStorage, error) {
 	switch configuration.Type {
 	case types.SQLStorage:
+		log.Info().Str("OCP storage type", configuration.Type).Send()
```

Comment:
"SQL storage type"

We should stop using "OCP", now that DVO is coming in the picture

Reply:
it was actually just SQL before, but now we make a difference in the logs between OCP and DVO storage

Reply:
ah, my bad. it's in the 'NewOCPRecommendationsStorage' function, so this log does make sense. Scratch this comment too =)